pages/learn/manage/device-statuses.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Device statuses
+excerpt: Determine the status and connectivity of a device
+---
+
+# Device statuses
+
+The application overview page shows the status of each device in the application.
+
+<img src="/img/common/main_dashboard/application_device_status.png" alt="Application device status" width="40%" >
+
+Each device can have one of the following statuses:
+
+| Status            | Description                                                                                  |
+|-------------------|----------------------------------------------------------------------------------------------|
+| Online            | The device is online and is connected to either the VPN or has recent API communications.    |
+| Configuring       | The device is applying a configuration update.                                               |
+| Updating          | The device is updating to a new application release.                                         |
+| Offline           | The device is offline and is not connected to VPN and has not any recent API communications. |
+| Post Provisioning | The device has been [provisioned][provisioned] but has not yet come online.                  |
+| Inactive          | The device has been [deactivated][deactivated].                                              |
```

Comment:
Just FYI that deactivation is not the only way to end up on this state.
A pre-provisioned device (with `balena device register`.) shows up as inactive until it first connects to the API.

Reply:
Perfect, I can link to the [preregistering section](https://www.balena.io/docs/learn/more/masterclasses/advanced-cli/#52-preregistering-a-device) of the masterclass for that too.